pkg/controller/sqs/queue/controller.go

```diff
@@ -138,13 +139,19 @@ func (e *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 	}
 
 	cr.SetConditions(xpv1.Creating())
-
-	_, err := e.client.CreateQueueRequest(&awssqs.CreateQueueInput{
+	resp, err := e.client.CreateQueueRequest(&awssqs.CreateQueueInput{
 		Attributes: sqs.GenerateCreateAttributes(&cr.Spec.ForProvider),
 		QueueName:  aws.String(meta.GetExternalName(cr)),
 		Tags:       cr.Spec.ForProvider.Tags,
 	}).Send(ctx)
-	return managed.ExternalCreation{}, errors.Wrap(err, errCreateFailed)
+
+	if err != nil {
+		return managed.ExternalCreation{}, errors.Wrap(err, errCreateFailed)
+	}
+	conn := managed.ConnectionDetails{
```

Comment:
nitpick: possibly you can set the URL in status and reuse `GetConnectionDetails` function here as well.